### PR TITLE
chore: add gemfile production group

### PR DESCRIPTION
**Pull request by JuanVqz**
do we need a honeybadger gem on development?

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,13 +28,16 @@ group :test do
   gem "capybara"
 end
 
+group :production do
+  gem "honeybadger", "~> 4.0"
```
Comment (vamsipavanmahesh) [inline Gemfile:32]: Why do we want to do this? IMHO except local machine all errors should be logged once the code goes on cloud, even on non production envs such as UAT, Staging, development

Comment (joemasilotti) [reply Gemfile:32]: What's the difference between development and local machine?

The site only has dev (local) and production right now. So moving this gem to the production-only group make sense to me.

**Comment by joemasilotti** [comment]
> do we need a honeybadger gem on development?

Nope, I don't believe we do. It was only there to test the error reporting during setup.

Can you please confirm that when you move the gem to production, and run `bundle install`, that raising an error in development doesn't interfere at all with Honeybadger?

File: Gemfile (hunk 1 above)
Comment (vamsipavanmahesh) [reply Gemfile:32]: local is basically developers machine, dev environment could be a hosted environment, staging/uat could be just before prod

dev environment is usually unstable, branch testing etc can happen there

staging is stable

I don't know what other environments are present before prod, but if there are any, ideally it's good if honey badger could catch them before going to prod

**Comment by joemasilotti** [comment]
On second thought, and after reading @vamsipavanmahesh's comments, I'm leaning towards keeping this in all environments.

Keeping Honeybadger in all environments will make creating a staging environment one step easier. I'm going to close this PR, but thanks for sparking the discussion!

**Comment by JuanVqz** [comment]
I had a long weekend, I could check today.

I think we still have another option when we need staging.

```
group :production, :staging do
 gem...
end
```

**Pull request closed without merging**